src/renderer/components/_features/[workspace]/editor/data-type/index.tsx

```
@@ -68,14 +79,23 @@
+                onChange={handleChange}
+                onBlur={handleBlur}
+                id='data-type-name'
+                aria-label='data-type-name'
+                className='h-full w-full bg-transparent px-3 text-start font-caption text-xs text-neutral-850 outline-none dark:text-neutral-100'
+              />
+            ) : (
+              <p
+                aria-label='Data type name'
+                className='h-full w-full bg-transparent px-3 text-start font-caption text-xs text-neutral-850 outline-none dark:text-neutral-100'
+                onClick={handleStartEditing}
+                dangerouslySetInnerHTML={{ __html: extractSearchQuery(editorContent?.name || '', searchQuery) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)